mbuild/__init__.py

```diff
@@ -1,5 +1,6 @@
 from mbuild.box import Box
 from mbuild.coarse_graining import coarse_grain
+from mbuild.reverse_mapping import reverse_map
```

Comment:
It would make the code a little more condensed if the routines in `reverse_mapping` were moved to `coarse_graining.py`. I know here we're doing the opposite of coarse-graining, but it makes sense to me to have both the coarse-graining and reverse mapping routines housed within a single file. @ctk3b may have an opinion on this.

Reply:
Okay, I didn't touch this refactoring in the latest commit, but receptive to changing it in the future. It might be good to group them because they're related functions and might reduce the number of different files

Reply:
I don't have a strong preference - they both seem to make sense.